test/box-py/args.result

```diff
@@ -152,3 +156,28 @@ arg[2] => 2
 arg[3] => 3
 arg[4] => --help
 
+-b
+Save Tarantool bytecode: tarantool -b[options] input output
```

Comment:
Very useful help message for `-b` option! Thanks!

(It was very hard to figure out list of `-j` or `-b` suboptions supported, as `luajit` usage help was not very helpful.)

Reply:
Oh, I was too much optimistic here, and apparently that was from script itself, and we still do not add any useful information in the tarantool usage screen. :(

Reply:
TBH, I don't get what and where do you want to see.